README.md

```diff
@@ -1,2 +1,69 @@
-# TalentForgery
-Compose training tutorials, create questionnaire, train your staff, create a knowledge sharing work environment
+
+# Minon Mentor
+Welcome to our innovative web app that revolutionizes training and knowledge sharing within your organization. With our powerful features, you can effortlessly compose captivating training tutorials, craft engaging questionnaires, empower your staff through effective training, and foster a collaborative work environment for seamless knowledge exchange.
+
+
+## Get Started
+### Client
+- ```npm i``` to install the required node modules
+- ```npm test``` to run the client tests
```

Comment:
all of the tests on the frontend will fail, maybe advise using npx cypress open on the root directpry instead